src/routes/PatronBlockContainer.js

```diff
@@ -26,6 +26,11 @@ class PatronBlockContainer extends React.Component {
         path: 'manualblocks/%{activeRecord.blockid}',
       }
     },
+    blockTemplates: {
+      type: 'okapi',
+      records: 'manualBlockTemplates',
+      path:'manual-block-templates',
```

Comment:
Without an explicit `limit` clause, you get the default limit of 10. Even if that likely meets your needs, it may be worth bumping to 100 or 1000 to be more certain it meets all tenants' needs.